includes/wikia/WikiaUpdater.php

```diff
@@ -33,6 +33,7 @@ public static function update( DatabaseUpdater $updater ) {
 			array( 'addTable', 'page_wikia_props', $ext_dir . '/wikia/ImageServing/sql/table.sql', true ),
 			array( 'addTable', 'wall_history', $ext_dir . '/wikia/Wall/sql/wall_history_local.sql', true ),
 			array( 'addTable', 'wall_related_pages', $ext_dir . '/wikia/Wall/sql/wall_related_pages.sql', true ),
+			array( 'addTable', 'article_comments', $ext_dir . '/wikia/ArticleComments/sql/article_comments.sql', true ),
```

Comment:
Actually I think it might be better to do a kosher MediaWiki approach here, and register a `LoadExtensionSchemaUpdates` hook handler instead. Then we can run update.php only for wikis with Article Comments or Blogs enabled, and only create the table when necessary.

We'll also need to handle cases where an user enables comments or blogs via Wiki Features page—we can listen on `WikiFeatures::afterToggleFeature` event for that.